app/controllers/vitals_controller.rb

```diff
@@ -76,6 +77,11 @@ def chart
   end
 
   private
+
+    def authorize_resource
```

Comment:
See above comment on before_action of why I feel this method might not be necessary